tests/PySys/mqtt_port/mqtt_port_change_connection_fails/pysystest.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pysystest type="auto">
+	<description>
+		<title>
+			Validates the mqtt port change that fails mqtt clients
+			The tedge mqtt pub/sub fail to connect to broker
```

Comment:
I don't really understand the title

Reply:
I added title same as the test folder name